paperweight-lib/src/main/kotlin/tasks/ApplyCraftBukkitPatches.kt

```diff
@@ -34,27 +34,26 @@ import org.gradle.api.file.DirectoryProperty
 import org.gradle.api.file.RegularFileProperty
 import org.gradle.api.provider.Property
 import org.gradle.api.provider.ProviderFactory
-import org.gradle.api.tasks.Input
-import org.gradle.api.tasks.InputDirectory
-import org.gradle.api.tasks.InputFile
-import org.gradle.api.tasks.Optional
-import org.gradle.api.tasks.OutputDirectory
-import org.gradle.api.tasks.TaskAction
+import org.gradle.api.tasks.*
 
+@CacheableTask
 abstract class ApplyCraftBukkitPatches : ControllableOutputTask() {
 
     @get:InputFile
+    @get:PathSensitive(PathSensitivity.RELATIVE)
```

Comment:
SHould use ` @get:PathSensitive(PathSensitivity.NONE)` for individual files. The only thing that matters is the field contents.

Reply:
yeah that's a copy paste mistake, did that on the other file property